comixed-webui/src/assets/i18n/pt/library.json

```diff
@@ -101,6 +102,14 @@
       "effect-failure": "Falhou o começo da conversão.",
       "effect-success": "A começar a conversão de {count, plural, =1{um comic} other{# comics}} para {archiveType}."
     },
+    "edit-multiple-comics": {
+      "confirm-message": "Are you sure you want to edit the details for {count, plural, =1{one comic} other{# comics}}?",
+      "confirm-title": "Edit Details For {count, plural, =1{One Comic} other{# Comics}}?",
+      "effect-failure": "Failed to edit multiple comics.",
+      "effect-success": "Multiple comics edited.",
+      "header": "WARNING: The values entered here will be applied to {count, plural, =1{the selected comic} other{all # selected comics}}.",
+      "title": "Edit The Details For {count, plural, =1{One Comic} other{# Comics}}..."
+    },
```

Comment:
Everything looks fine but if you want to update PT language here it goes:

"edit-multiple-comics": "Editar Dados dos Comics Selecionados",
 
"edit-multiple-comics": "Tem a certeza que quer editar os dados de {count, plural, =1{um comic} other{# comics}}?",
"confirm-title": "Editar dados de {count, plural, =1{Um Comic} other{# Comics}}?",
"effect-failure": "Falhou ao editar varios comics.",
"effect-success": "Varios comics editados.",
"header": "ATENÇÃO: Os valores introduzidos aqui serão aplicados ao{count, plural, =1{ comic selecionado} other{s # comics selecionados}}.",
"title": "Editar Dados De {count, plural, =1{Um Comic} other{# Comics}}..."